src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -300,7 +309,6 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
           <TokenSelector onClick={listItems?.length > 1 ? openSearchList : undefined} center={hideAmount}>
             <TokenIconContainer>
               <TokenIcon icon={selectedItem.icon} symbol={selectedItem.label} size="big" />
-              {listItems?.length > 1 && <TokenListIcon Component={ChevronRightIcon} />}
             </TokenIconContainer>
```

Comment:
Is the token still meant to be clickable?
![image](https://user-images.githubusercontent.com/18649057/169622035-2257c1c8-e06c-4942-b109-eb4f78ec0cf9.png)

Reply:
yes - for ppl who have muscle-memory (i checked with b)